accountClient/accountClient.go

```diff
@@ -46,6 +46,35 @@ func (c *E3dbAccountClient) InternalGetClientAccount(ctx context.Context, client
 	return result, err
 }
 
+func (c *E3dbAccountClient) RegisterClient(ctx context.Context, params ClientRegistrationRequest) (*ClientRegistrationResponse, error) {
+	var result *ClientRegistrationResponse
+	path := c.Host + "/" + AccountServiceBasePath + "/e3db/clients/register"
```

Comment:
We have a client-service client to query the client service, purely written for testing purposes

Reply:
Eventually the account service version of this endpoint will get removed -- it will probably be a little while before everything is relying on the new endpoint, but in general we should not write any new code relying on the deprecated endpoints. If we need this for compatibility, then it's fine, but if this is new testing functionality then we should either use the client service client, or this should hit the client service endpoint.

I may not be understanding what this is, too, so feel free to correct me if need be :)

Reply:
Just an added layer for me to make sure the registration implementation was working. We had the from account service to client service so I wanted to make sure callers of the account service would get back expected results. For the future client service should be used. I updated this with a comment that it should be used for testing, and I can remove this method now or later if you like